### PR TITLE
Go version update

### PR DESCRIPTION
**Pull request by adarsh-dell**
# Description
Go version update

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/885|

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Tested with PowerStore.

### CODE REVIEW

**Review by AkshaySainiDell — APPROVED** [review]
lgtm

**Review by suryagupta4 — APPROVED** [review]

File: gonvme_utils.go [hunk 1]
```diff
@@ -18,9 +18,10 @@ package gonvme
 
 import (
 	"encoding/json"
-	log "github.com/sirupsen/logrus"
 	"regexp"
 	"strings"
+
```
Comment (prablr79) [inline gonvme_utils.go:23]: remove the extra line

File: gonvme_test.go [hunk 2]
```diff
@@ -19,10 +19,11 @@ package gonvme
 import (
 	"encoding/json"
 	"fmt"
-	log "github.com/sirupsen/logrus"
 	"io/ioutil"
 	"strings"
 	"testing"
+
```
Comment (prablr79) [inline gonvme_test.go:25]: remove the extra line

Comment (adarsh-dell) [reply gonvme_test.go:25]: Thanks @prablr79 for the feedback. Actually it is automatically getting added by the go formatter tool.

**Review by prablr79 — APPROVED** [review]

**Pull request merged**